cci-stats/pkg/db/db.go

```diff
@@ -0,0 +1,211 @@
+package db
```

Comment:
I'm not sure how important this service is, but there's a lot of untested code in here and I feel the need to call it out.

Reply:
All this service does is populate the stats on http://go/ci-dashboard. Bugs in this code just make the stats show up wrong, and the entire project is just a hair over 500 SLOC so I didn't think it was worth investing much time in detailed unit testing particularly since the majority of the logic reads from the CCI API and would require mocking.